sbncode/CAFMaker/FillReco.cxx

```diff
@@ -289,6 +289,33 @@ namespace caf
   }
 
 
+  void FillSliceCRUMBS(const sbn::CRUMBSResult *crumbs,
+                       caf::SRSlice& slice,
+                       bool allowEmpty) {
```

Comment:
allowEmpty is now unused. Either add an `if(!crumbs && !allowEmpty) abort("no crumbs! (better text please...)");` or remove the argument.

Reply:
I have removed it for now. I noted, however, whilst doing this that for almost every function in `FillReco` this argument exists and is unused... something for a separate PR!